### ISSUE TITLE
JRuby support

### ISSUE THREAD

**Issue by dblock** [danger/danger#394]
Looks like danger use redcarpet only for markdown parsing, this not possible with jruby. 

Workaround for Gemfiles:

```ruby
unless RUBY_PLATFORM == 'java'
  gem 'danger', '~> 2.0', require: false
end
```

**Comment by orta** [comment]
Hrm, yeah, I'm not sure that we can live without redcarpet, it's the only markdown rendered that does github flavoured markdown right. Think we should document this 👍

**Comment by Bugagazavr** [comment]
As i know kramdown can support GFM http://kramdown.gettalong.org/parser/gfm.html

**Comment by LeFnord** [comment]
@orta … yeap I would also prefer `kramdown` over `redcarpet`

**Comment by orta** [comment]
Shipped as 2.1.1 👍
